app/models/authenticated_session.rb

```diff
@@ -45,6 +45,16 @@ def save
     end
 
     false
```

Comment:
I know this is not from this PR, but I think this `false` never did anything -- the if/else above will always return before this line is executed.

Reply:
Oh that's a good point!